prydaen-favors.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+    Author: Damiza Nihshyde
+    Version: 2.0
+    Thanks for assistance:  Binu, Corgar, Mahtra, Bellz
+    Errors?  DM Obsidian#1345
+
+    Prydaens only, no Furless!
+    Set favor_god to "Triquetra" without the quotes.
+
+    Must have at least 65 ranks of perception to get to the Prydaen boulder.
+
+    **WARNING**
+    - Be sure your health_threshold is not going to cause you to log off.
+      You will have significant blood loss doing Eu's offering.
+    
+    changelog:
+        2.0 (2022-01-30):
+            **Functional with checkfavors or by itself (waiting on merge)
+            Purchases rat-pelt leather from Outfitting Society to use as an offering
+=end
+
+custom_require.call(%w[common common-travel common-money])
+
+class Prydaen_Favors
+  include DRC
+  include DRCT
+
+  def initialize
+    unless DRStats.race == 'Prydaen'
```

Comment:
The general feedback has been that `unless` can be a little confusing, so our convention has been to use `if` when possible.

```suggestion
    if DRStats.race != 'Prydaen'
```

Reply:
Ooooh, okies.